keyboards/star75/keymaps/default/readme.md

```diff
@@ -0,0 +1,16 @@
+# Default Star75 Layout
+
+This is the default layout of Star75 PCB.
+
+Note that the optional encoder is disabled by default.
+
+rules.mk OPTIONS
+
+ENCODER_MAPPING_ENABLE = yes - turns on the optional encoder
```

Comment:
```suggestion
`ENCODER_MAP_ENABLE = yes` - turns on the optional encoder
```

Also, note that the encoder feature needs to be enabled, as well, for this to work. Eg, `ENCODER_ENABLE = yes`